src/components/App.jsx

```diff
@@ -1,14 +1,183 @@
-import React from 'react'
+import React, { Component } from 'react'
```

Comment:
There are a handful of small things below that are linter errors. `npm run lint` will run the linter, which will find these issues. Fixing all linter errors is highly recommended.

In addition, the linter for this project is setup according to some pretty common patterns that developers use for JavaScript around the world, so it's a decent guide on how folks tend to do things.

Reply:
done! great to know!!